src/components/App.js

```diff
@@ -25,26 +25,29 @@ const App = ({options, selectedLanguage, changeLanguage, word, changeWord, langu
     }}
 
   return (
+
     <div className="App">
-      <div>
-        <h1>Select Your Language!</h1>
+      <div className="title-container">
+        <span className="title" >CONJUGATOR</span>
       </div>
-
+      <div className='content-container'>
+        <p className="instruction_text">Select Your Language!</p>
       <Dropdown options={languageList} onChange={(languageOption) => {changeLanguage(languageOption.value)}} value={selectedLanguage} placeholder="Please Select Language" />
       {optionListForm} {optionBulletForm}
-      <div>
-        <input type="text" name="word" placeholder="Please input word" value={word} onChange={(e) => {
+        <p className="instruction_text">Type Your Word</p>    
```

Comment:
be consistent with naming convention. use dashes instead of underscore: `instruction-text`